docs/Populate.md

```diff
@@ -0,0 +1,118 @@
+# Populate Exchange
+
+`populate` is an exchange for auto-populating fields in your mutations.
+
+## How to use
+
+As with any exchange, add this to your client (before your cache).
+
+```tsx
+import { populateExchange } from '@urql/exchange-graphcache';
+
+const client = createClient({
+  // ...
+  exchanges: [dedupExchange, populateExchange, cacheExchange, fetchExchange],
+})
+```
+
+## Example usage
+
+Consider the following queries which have been requested
+
+
+```graphql
+# Query 1
+{
+  todos {
+    id
+    name
+  }
+} 
+
+# Query 2
+{
+  todos {
+    id
+    createdAt
+  }
+}
+```
+
+Now consider the following mutation which returns a todo
+
+```graphql
+# Without populate
+mutation addTodo(id: ID!) {
+  addTodo(id: $id) {
+    id        # To update Query 1 & 2
+    name      # To update Query 1
+    createdAt # To update Query 2
+  }
+}
+```
+
+By using the `populate` exchange, you no longer need to manually specify the selection set required to update your other queries.
+
+```graphql
+# With populate
+mutation addTodo(id: ID!) {
+  addTodo(id: $id) @populate
+}
+```
+
+> Note: The above two mutations produce an identical GraphQL request.
+
+### Choosing when to populate
+
+You may not want to populate your whole mutation response. In order to reduce your payload, pass populate lower in your query.
```

Comment:
```suggestion
You may not want to populate your whole mutation response. In order to reduce your payload, can pass populate lower in your query. This will depend on how much data your UI needs to reflect after a mutation.
```

Reply:
Not sure I agree with this one

> In order to reduce your payload, can pass populate lower in your query

Adding "can" makes this grammatically incorrect

> This will depend on how much data your UI needs to reflect after a mutation.

I don't see what value this is adding - there are many reasons you might only want to populate a given field, this implies otherwise.

Reply:
Oops, that first one was accidental 😅 the second one is more of a placeholder, we should list out the use cases shortly after or before, but realistically I didn’t want to write a whole new section, so this is more of a temporary addition